packages/chaire-lib-frontend/src/components/routers/PublicRoute.tsx

```diff
@@ -21,7 +21,7 @@ type PublicRouteProps = RouteProps & {
     config?: { [key: string]: unknown };
 } & PropsWithChildren;
 
-const PublicRoute = ({ component: Component, children, ...rest }: PublicRouteProps) => {
+const PublicRoute = ({ component: Component, ...rest }: PublicRouteProps) => {
```

Comment:
I think the idea here was to remove the children from the props that you pass to the component (I'm not sure why), but the effect of this is that now Component will receive the children if there are any.

To fix this warning, you can do `children: _children` to rename the prop that is called `children` (I think)